spanner/tests/system/test_system.py

```diff
@@ -282,6 +282,20 @@ def test_create_database(self):
             for database in Config.INSTANCE.list_databases()]
         self.assertIn(temp_db_id, database_ids)
 
+    def test_table_not_found(self):
+        table = 'yTable'
+        temp_db_id = 'temp_db' + unique_resource_id('_')        
+        temp_db = Config.INSTANCE.database(temp_db_id, ddl_statements=[
+            'CREATE TABLE MyTable ('
+            '    Id      STRING(36) NOT NULL,'
+            '    Field1  STRING(36) NOT NULL'
+            ') PRIMARY KEY (Id)',
+            'CREATE INDEX IDX ON {table} (Field1)'.format(table)
+        ])
+        with self.assertRaisesRegexp(GrpcRendezvous,
+                                     'Table not found: {table}'.format(table)):
```

Comment:
Rather than use a regexp, can we just assert the entire error message? (Unless it's got "temporary" data in there like some opaque ID.)